### PR TITLE
Srinter 749

### PR DESCRIPTION
**Pull request by soda736**
I have enabled the media module for adding other media files. I added the Inplacenavigation block that is provided by the origins common module. I had to add ids/classes to page.html.twig and the inplace nav block as the javascript was looking for them. 
I have changed the publication search page name to something more descriptive.
Also added facet pretty paths to tidy up the URL's when facets are selected.

### CODE REVIEW

File: config/sync/system.site.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ slogan: ''
 page:
   403: ''
   404: ''
-  front: /node/
+  front: /node/7
```
Comment (ooneill547) [inline config/sync/system.site.yml]: I think this should be kept as /node in git until we are sharing a database with the content

Comment (soda736) [reply config/sync/system.site.yml]: Ah right no worries I'll change that back.

File: config/sync/system.performance.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ fast_404:
   exclude_paths: '/\/(?:styles|imagecache)\//'
   html: '<!DOCTYPE html><html><head><title>404 Not Found</title></head><body><h1>Not Found</h1><p>The requested URL "@path" was not found on this server.</p></body></html>'
 js:
-  preprocess: true
+  preprocess: false
```
Comment (omahm) [inline config/sync/system.performance.yml]: Change this back to true and disable JS preprocessing in your settings.local.php file. Give me a shout if you need a hand setting this up.

File: web/themes/custom/cybersecurity/templates/block/block--inpagenavigationblock.html.twig [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{#
```
Comment (omahm) [inline web/themes/custom/cybersecurity/templates/block/block--inpagenavigationblock.html.twig:1]: This is pretty much the core (stable) block template... I'd create a default block template using this markup and remove this template.

**Review by omahm — CHANGES_REQUESTED** [review]
Looks grand. Just 1 major issue with turning JS preprocessing off, that would introduce a performance cull if it ever deployed to Prod.

File: config/sync/system.performance.yml (hunk 2 above)
Comment (soda736) [reply config/sync/system.performance.yml]: Turned the JS preprocess back on.

**Pull request merged**